test/src/test/resources/fixtures/deep renames/deep-rename-not-asked-for-with-unions.yml

```diff
@@ -22,7 +22,7 @@ overallSchema:
 # language=GraphQL
 underlyingSchema:
   Issues: |
-    union HasName = Issue | Edible
+    union HasName = Issue | Edible | Troll | User
```

Comment:
Honestly not sure why these tests are set up like this… this is not desirable behavior.

Reply:
Shall we remove them or replace them with new tests using the new format?

Reply:
I think I'll just leave them, just not sure why the overall schema had more members to begin with…